src/ledger/LedgerManagerImpl.cpp

```diff
@@ -348,41 +348,35 @@ LedgerManagerImpl::loadLastKnownLedger(bool restoreBucketlist,
 
     releaseAssert(latestLedgerHeader.has_value());
 
-    // Step 3. Restore BucketList if we're doing a full core startup
-    // (startServices=true), OR when using BucketListDB
-    if (restoreBucketlist || mApp.getConfig().isUsingBucketListDB())
```

Comment:
I'm not 100% sure this guard can be dropped -- is it always true?

Reply:
Ah yeah this is a little premature since this doesn't entirely drop SQL.